### ISSUE TITLE
Shorted outputs don't work in the presence of split nodes

### ISSUE THREAD

**Issue by isovector** [nturley/netlistsvg#110]
In the following netlist, `A` and `B0` both should be connected to the output:

```json
{"modules":{"module":{"ports":{"A":{"direction":"input","bits":[0]},"B":{"direction":"input","bits":[0,1]},"Y":{"direction":"output","bits":[0]}},"cells":{}}}}
```

but they are not! Instead, `B` is floating:

![2021-07-08-172919_180x148_scrot](https://user-images.githubusercontent.com/307223/125006243-7f9d9900-e012-11eb-9aaa-8e486499a675.png)

A more correct rendering would be something like this:

![correct](https://user-images.githubusercontent.com/307223/125006480-14a09200-e013-11eb-8f8d-443bd61135d9.png)

---

Interestingly removing the second wire from `B` properly connects everything:

```json
{"modules":{"module":{"ports":{"A":{"direction":"input","bits":[0]},"B":{"direction":"input","bits":[0]},"Y":{"direction":"output","bits":[0]}},"cells":{}}}}
```

![2021-07-08-172944_182x184_scrot](https://user-images.githubusercontent.com/307223/125006286-99d77700-e012-11eb-887c-2096c24eb0c4.png)

Thus, this appears to be a bug at the intersection of split nodes and shorts.

**Comment by nturley** [comment]
Good catch! When I added hyper edge support I did not consider splits and joins and vice versa.
